### PR TITLE
docs(azblob): correct auth to Azure storage service

### PR DESCRIPTION
**Pull request by librucha**
I corected documentation for upload blobs to azure Storage Service from [used library](https://github.com/google/go-cloud/blob/53ccd8db26a14fd2a40bfcb28af1a1ceba4663f4/blob/azureblob/azureblob.go#L30).

### CODE REVIEW

File: www/docs/customization/blob.md [hunk 1]
```diff
@@ -15,70 +15,70 @@ blobs:
...
-    # Those files will have their contents pass through the template engine,
-    # and its results will be uploaded.
-    #
-    # Since: v1.17 (pro)
-    # This feature is only available in GoReleaser Pro.
-    # Templates: allowed
-    templated_extra_files:
-      - src: LICENSE.tpl
-        dst: LICENSE.txt
+    - provider: azblob
```
Comment (caarlos0) [inline www/docs/customization/blob.md:18]: this is wrong, the `-` is in the line 12

**Review by caarlos0 — APPROVED** [review]
lgtm except for the one comment

File: www/docs/customization/blob.md (hunk 1 above)
Comment (librucha) [reply www/docs/customization/blob.md]: Yes is masked there. :D Sorry

**Comment by caarlos0** [comment]
thank you! 🙏

**Pull request merged**